src/main/java/wooteco/subway/exception/SubwayException.java

```diff
@@ -0,0 +1,22 @@
+package wooteco.subway.exception;
+
+import org.springframework.http.HttpStatus;
+import wooteco.subway.exception.dto.ErrorResponse;
+
+public class SubwayException extends RuntimeException {
+    HttpStatus httpStatus;
+    ErrorResponse errorResponse;
```

Comment:
```suggestion
    private final HttpStatus httpStatus;
    private final ErrorResponse errorResponse;
```

접근제어자를 생략하신 것은 의도된 내용일까요!?

Reply:
허업 ㅠ 실수입니당 꼼꼼한 피드백 감사드려요 ☺️ 바로 수정하겠습니당!!!!!!